src/Raven.Client/Documents/Queries/Explanation/Explanations.cs

```diff
@@ -6,6 +6,8 @@ public class Explanations
     {
         private Dictionary<string, string[]> _explanations;
 
+        public bool ShouldBeIncluded { get; set; } = false;
```

Comment:
in QueryTimings we made it internal (#14048). Why here it should be public? Also isn't false the default value for bool?

Reply:
Ok